src/hooks/index.tsx

```diff
@@ -73,15 +73,10 @@ export async function getFileContent(
   const { repo, owner, path, fileRef } = params;
   const branch = fileRef || "HEAD";
 
-  const apiUrl = `https://raw.githubusercontent.com/${owner}/${repo}/${branch}/${path}`;
-  const res = await fetch(
-    apiUrl,
-    PAT
-      ? {
-          headers: PAT ? { Authorization: `token ${PAT}` } : {},
-        }
-      : {}
-  );
+  const apiUrl = `https://api.github.com/repos/${owner}/${repo}/contents/${path}?ref=${branch}`;
+  const res = await fetch(apiUrl, {
+    headers: PAT ? { Authorization: `token ${PAT}` } : {},
+  });
```

Comment:
I think you need to unpack the `content` field from the response and base64-decode it, as we do in `getFileContent` in `blocks`. Or maybe there is a way to get raw content by giving a content-type header, see https://docs.github.com/en/rest/reference/repos#custom-media-types-for-repository-contents